File: official/docs/python/v7/addresses/verify-failure.py
import easypost

easypost.api_key = "EASYPOST_API_KEY"

address = easypost.Address.create(
    street1="UNDELIVERABLE ST",
    city="SAN FRANCISCO",
    state="CA",
    zip="94104",
    country="US",
    company="EasyPost",
    phone="415-123-4567",
)

print(address)

File: official/docs/php/v6/addresses/verify-failure.php
<?php

$client = new \EasyPost\EasyPostClient('EASYPOST_API_KEY');

$address = $client->address->create([
    'street1' => 'UNDELIVERABLE ST',
    'city'    => 'San Francisco',
    'state'   => 'CA',
    'zip'     => '94104',
    'country' => 'US',
    'company' => 'EasyPost',
    'phone'   => '415-123-4567'
]);

echo $address;
